RecoLocalCalo/HcalRecProducers/python/M0defaultFullRange_cff.py

```diff
@@ -0,0 +1,19 @@
+# Set M0 as default HCAL reconstruction over full HBHE Digi range
+# To be used in cmsDriver command: 
+# --customise RecoLocalCal/HcalRecProducers/M0defaultFullRange_cff.customiseM0
+
+import FWCore.ParameterSet.Config as cms
+def customiseM0(process):
```

Comment:
we usually recommend(ed) to 
- put customization methods to the [subsystem]/Configuration package; it would be RecoLocalCalo/Configuration/python in this case
- have no _cff in the file name where just python methods/functions are defined

The name either of the file or the method seem a bit cryptic to me; looking at the patterns of other custom methods, I'd suggest something like RecoLocalCalo/Configuration/python/customiseHBHEreco.py for the file name and `def hbheUseM0FullRange` for the method name, perhaps even `def hbheUseM0FullRangePhase1` because this method will not work in phase-0 context

Reply:
Done.  Worth it doing the same move in PR for 12_1 ?